pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -129,6 +129,12 @@ in
     ];
   };
 
+  pg_query = attrs: {
+    postPatch = ''
+      ln -s ${libpg_query}/lib ${libpg_query.name}
```

Comment:
the gem version in your project does not match the version on github: https://github.com/lfittl/pg_query/blob/v0.9.2/ext/pg_query/extconf.rb